### PR TITLE
Remove OPTIGA-Trust-E-Security-Controller

### PR DESCRIPTION
**Pull request by ayushev**
Remove the OPTIGA-Trust-E-Security-Controller as being deprecated, also the product is discontinued

### CODE REVIEW

**Comment by per1234** [comment]
Hello @ayushev. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

**Review by MatteoPologruto — APPROVED** [review]

**Comment by MatteoPologruto** [comment]
Done:
```
2023/01/25 17:08:58 Loaded 5513 libraries from DB
Removing OPTIGATrustE
warning: Library Git clone folder /home/ubuntu/workspace/gitclones/github.com/Infineon/OPTIGA-Trust-E-Security-Controller not present
Success!
```

**Pull request merged**
